robot.py

```diff
@@ -30,7 +28,7 @@ def createObjects(self):
             drive_talon=ctre.TalonSRX(2),
             x_pos=x_dist,
             y_pos=y_dist,
-            drive_free_speed=Robot.module_drive_free_speed,
+            drive_free_speed=SwerveModule.DRIVE_FREE_SPEED,
```

Comment:
Why is this being passed in here?

Reply:
Actually it should be a default argument no being passed in, right?

Reply:
Or not even as an argument